starter-code/classes/__tests__/vehicle.test.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const VehicleConstructor = require('../vehicle-constructor.js');
+const { Car, Motorcycle } = require('../vehicle-class.js');
+
+let types = ['Constructor', 'Class'];
+
+describe('Vehicles', () => {
+
+  describe('Car', () => {
+
+    function getCar(type) {
+      switch (type) {
+        case 'Constructor':
+          return new VehicleConstructor.Car('foo');
+        case 'Class':
+          return new Car('Nissan');
```

Comment:
same as the comment above you can do the same thing on line 15 with line 17 but for classes. (you don't have to take these suggestions they are up to you.)